plyer/facades/notification.py

```diff
@@ -46,7 +46,7 @@ class Notification:
     '''
 
     def notify(self, title='', message='', app_name='', app_icon='',
-               timeout=10, ticker='', toast=False):
+               timeout=10, ticker='', toast=False, **kwargs):
```

Comment:
I would much rather that all specific arguments be specified.

Reply:
Done!